src/webgl.js

```diff
@@ -278,11 +278,10 @@ function init_conv1_1_program(gl) {
     out1 = vec4(0.0, 0.0, 0.0, 0.0);
     out2 = vec4(0.0, 0.0, 0.0, 0.0);
     out3 = vec4(0.0, 0.0, 0.0, 0.0);
-    float outX = float(gl_FragCoord[0]);
-    float outY = float(gl_FragCoord[1]);
+    float outX = float(gl_FragCoord.x);
+    float outY = float(gl_FragCoord.y);
```

Comment:
Curious if the `float()` is even required? Those values in the vec should already be floats.

Reply:
I was thinking that too, but I always wonder when I see something like this if maybe there is some clever trick that I am not catching on to. I'll update this in my clean up branch.